functions/json.func.php

```diff
@@ -31,7 +31,7 @@ function toJSON( $allPairData ) {
 /**
  * Creates the compatibility graph for all the data in database
  *
- * @param  json $jsonData     contains all the data from database in json format
```

Comment:
This is not `string` it is `json` type so, it can be converted to associative array in PHP (i.e, dict in python) for parsing

Reply:
- okaey